docs/changelog/98204.yaml

```diff
@@ -6,8 +6,8 @@ issues: []
 highlight:
   title: Enable parallel knn search across segments
   body: |-
-    Elasticsearch has until now performed search sequentially across the
+    Elasticsearch has, until now, performed search sequentially across the
     segments within each shard. This change makes knn queries faster on shards
-    that are made of more than one segment, by rewriting and collecting each
+    that are made of more than one segment by rewriting and collecting each
```

Comment:
FYI: I modified the phrasing in this `.yaml` file ([as per the instructions](https://docs.elastic.dev/elasticsearch-team/release/release-management#curate-the-notes)).